.goreleaser.yaml

```diff
@@ -71,12 +71,9 @@ builds:
     goarch:
       - amd64
       - arm64
-      - arm
```

Comment:
there was discussion around this in the original PR and keeping the arm support: https://github.com/guacsec/guac/pull/910#discussion_r1303123801

Reply:
Judging from the ent issue, it seems like it will not work. I'll create an issue to investigate.

Reply:
#1206